elastic/kibana
# [SecuritySolution] Relocate endpoint tests

## Summary

https://github.com/elastic/kibana/issues/181683

This PR moves 

1. x-pack/test/security_solution_endpoint_api_int to `x-pack/test/security_solution_api_integration/test_suites/security_solution_endpoint_api_int`
2. x-pack/test/security_solution_endpoint to `x-pack/test/security_solution_api_integration/test_suites/security_solution_endpoint`
3. x-pack/test/timeline to `x-pack/test/security_solution_api_integration/test_suites/investigation/timeline`

### To test:

1. ```cd x-pack/test/security_solution_api_integration```

2. ```node ../../../scripts/functional_tests_server.js --config ./test_suites/security_solution_endpoint/configs/serverless.endpoint.config.ts```

Once the server is launched (you might need Docker to run the serverless tests), open another terminal, go to the same path, and execute the command appears in the original:

The command should look like: ```node ../../../scripts/functional_test_runner --config=test_suites/security_solution_endpoint/configs/serverless.endpoint.config.ts```
